core/services/keeper/common.go

```diff
@@ -4,17 +4,20 @@ import (
 	"math/big"
 	"time"
 
+	evmtypes "github.com/smartcontractkit/chainlink/core/chains/evm/types"
+
 	"github.com/ethereum/go-ethereum/common"
 
-	"github.com/smartcontractkit/chainlink/core/chains/evm/types"
 	"github.com/smartcontractkit/chainlink/core/gethwrappers/generated/keeper_registry_wrapper1_1"
 	"github.com/smartcontractkit/chainlink/core/gethwrappers/generated/keeper_registry_wrapper1_2"
 	"github.com/smartcontractkit/chainlink/core/gethwrappers/generated/keeper_registry_wrapper1_3"
+	"github.com/smartcontractkit/chainlink/core/gethwrappers/generated/keeper_registry_wrapper2_0"
 )
 
-var Registry1_1ABI = types.MustGetABI(keeper_registry_wrapper1_1.KeeperRegistryABI)
-var Registry1_2ABI = types.MustGetABI(keeper_registry_wrapper1_2.KeeperRegistryABI)
-var Registry1_3ABI = types.MustGetABI(keeper_registry_wrapper1_3.KeeperRegistryABI)
+var Registry1_1ABI = evmtypes.MustGetABI(keeper_registry_wrapper1_1.KeeperRegistryABI)
+var Registry1_2ABI = evmtypes.MustGetABI(keeper_registry_wrapper1_2.KeeperRegistryABI)
+var Registry1_3ABI = evmtypes.MustGetABI(keeper_registry_wrapper1_3.KeeperRegistryABI)
+var Registry2_0ABI = evmtypes.MustGetABI(keeper_registry_wrapper2_0.KeeperRegistryABI)
```

Comment:
don't think we need this here after the separate plugin repo

Reply:
Removed this one